### PR TITLE
Fix some cosmetics issues

### PR DESCRIPTION
**Pull request by simonxiaoss**

### CODE REVIEW

File: LISAv2-Framework.psm1 [hunk 1]
```diff
@@ -160,7 +161,7 @@ function Start-LISAv2 {
 			}
 
 			# Run test
-			$testController.RunTest($TestReportXml,$TestIterations,$false)
+			$testController.RunLoadedTestCases($TestReportXml, $TestIterations, $false)
```
Comment (simonxiaoss) [inline LISAv2-Framework.psm1:164]: In above code, there is a function "LoadTestCases". So here rename this function to "RunLoadedTestCases" makes more sense.

File: LISAv2-Framework.psm1 [hunk 2]
```diff
@@ -81,6 +81,7 @@ function Start-LISAv2 {
 			$maxDirLength = 32
 			$workingDirectory = (Get-Location).Path
 			if ($workingDirectory.Length -gt $maxDirLength) {
+				Write-LogWarn "The path of current working directory '$workingDirectory' is too long ($workingDirectory.Length)."
```
Comment (simonxiaoss) [inline LISAv2-Framework.psm1:84]: Just move the logging from helper function (Move-ToNewWorkingSpace) to here.

File: TestControllers/TestController.psm1 [hunk 3]
```diff
@@ -407,7 +407,7 @@ Class TestController
 		return $currentTestResult
 	}
 
-	[object] RunTestCase($VmData, $CurrentTestData, $ExecutionCount, $SetupTypeData, $ApplyCheckpoint) {
+	[object] RunOneTestCase($VmData, $CurrentTestData, $ExecutionCount, $SetupTypeData, $ApplyCheckpoint) {
```
Comment (simonxiaoss) [inline TestControllers/TestController.psm1:410]: In previous implementation, there are many functions with different purposes but have similar function name. For example, "RunTestCase", "RunTest", etc.
Here, those confusing function names are renamed with more specific names.

File: TestControllers/TestController.psm1 [hunk 4]
```diff
@@ -513,8 +511,8 @@ Class TestController
 		$this.TestSummary.UpdateTestSummaryForCase($CurrentTestData, $ExecutionCount, $currentTestResult.TestResult, $testRunDuration, $currentTestResult.testSummary, $VmData)
 
 		# Update junit report for current test case
-		$caseLog = ($currentTestResult.testSummary + (Get-Content -Raw "$CurrentTestLogDir\$global:LogFileName")).Trim()
```
Comment (simonxiaoss) [inline TestControllers/TestController.psm1:516]: rename "caseLog" to "testCaseLog"

File: TestControllers/TestController.psm1 [hunk 5]
```diff
@@ -557,24 +555,24 @@ Class TestController
...
-		foreach ($key in $this.SetupTypeToTestCases.Keys) {
-			$setupType = $key.Split(',')[0]
+		foreach ($setupKey in $this.SetupTypeToTestCases.Keys) {
+			$setupType = $setupKey.Split(',')[0]
 
 			$vmData = $null
 			$lastResult = $null
 			$tests = 0
 
-			foreach ($case in $this.SetupTypeToTestCases[$key]) {
```
Comment (simonxiaoss) [inline TestControllers/TestController.psm1:571]: rename "case" to "currentTestCase"

File: TestProviders/AzureProvider.psm1 [hunk 6]
```diff
@@ -96,7 +96,7 @@ Class AzureProvider : TestProvider
 		return @{"VmData" = $allVMData; "Error" = $ErrorMessage}
 	}
 
-	[void] DeleteTestVMs($allVMData, $SetupTypeData, $UseExistingRG) {
```
Comment (simonxiaoss) [inline TestProviders/AzureProvider.psm1:99]: Above we have "DeployVM". So here use "DeleteVM" makes more sense.

**Review by harz566 — APPROVED** [review]

**Review by lizzha — APPROVED** [review]

File: TestControllers/TestController.psm1 [hunk 7]
```diff
@@ -584,78 +582,83 @@ Class TestController
...
 
-					Write-LogInfo "$($case.testName) ended running with status: $($lastResult.TestResult)."
+					Write-LogInfo "$($currentTestCase.testName) ended running with status: $($lastResult.TestResult)."
 				}
 			}
 
 			# Delete the VM after all the cases of same setup are run, if DeployVMPerEachTest is not set
 			if ($this.TestCasePassStatus.contains($lastResult.TestResult) -and !($this.ResourceCleanup -imatch "Keep") -and !$this.DeployVMPerEachTest) {
-				$this.TestProvider.DeleteTestVMS($vmData, $this.SetupTypeTable[$setupType], $this.UseExistingRG)
+				Write-LogInfo "Delete deployed target machine if required ..."
```
[the remark this answers is not in the document]
Comment (simonxiaoss) [reply TestControllers/TestController.psm1]: Maybe not. The DeleteVM() maybe does nothing - for example, for Ready Platform :)

File: LISAv2-Framework.psm1 (hunk 2 above)
Comment (LiliDeng) [inline LISAv2-Framework.psm1:84]: ($workingDirectory.Length) => $($workingDirectory.Length)
Current output - The path of current working directory 'C:\jenkins-slave-executor-3\workspace\Azure\job-azure-cloudtest-manual' is too long (C:\jenkins-slave-executor-3\workspace\Azure\**job-azure-cloudtest-manual.Length**).

**Pull request merged**